vnext/Microsoft.ReactNative/Microsoft.ReactNative.vcxproj

```diff
@@ -812,6 +812,7 @@
     <Import Project="$(SolutionDir)packages\$(WinUIPackageName).$(WinUIPackageVersion)\build\native\$(WinUIPackageName).targets" Condition="Exists('$(SolutionDir)packages\$(WinUIPackageName).$(WinUIPackageVersion)\build\native\$(WinUIPackageName).targets')" />
     <Import Project="$(V8Package)\build\native\ReactNative.V8JSI.Windows.UWP.targets" Condition="Exists('$(V8Package)\build\native\ReactNative.V8JSI.Windows.UWP.targets') AND '$(UseV8)' == 'true'" />
     <Import Project="$(HermesPackage)\build\native\ReactNative.Hermes.Windows.targets" Condition="Exists('$(HermesPackage)\build\native\ReactNative.Hermes.Windows.targets')" />
+    <Import Project="$(SolutionDir)packages\Microsoft.Web.WebView2.$(WebView2Version)\build\native\Microsoft.Web.WebView2.targets" Condition="Exists('$(SolutionDir)packages\Microsoft.Web.WebView2.$(WebView2Version)\build\native\Microsoft.Web.WebView2.targets')" />
```

Comment:
who defines `WebView2Version`?

Reply:
winui nuget packages that include the wv2 support will require us to start referencing wv2 nuget here, regardless of whether the app uses wv2 or not. So:
1) there needs to be a `WebView2Version` set once wv2 comes to stable winui versions,
2) this check probably needs to validate whether WebView2Version is set in addition to the `Exists` condition